real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/HillCrestImuNode.java

```diff
@@ -228,4 +273,14 @@ public double convertQNToDouble(byte lsb, byte msb, int qn) {
 
     }
 
+    private int extractByteFromDataArray(int[] data, int offset) {
```

Comment:
The name says "extractByteFromDataArray" but it looks like we're plucking two bytes out.